storage/bootstrap/cache/services.php

```diff
@@ -0,0 +1,240 @@
+<?php return array (
```

Comment:
This file should not be provided. Please remove it from the PR.
(possibly add it to the `.gitignore`)